core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.html

```diff
@@ -1,9 +1,9 @@
-<div class="editor-wrapper" [style]="customStyles">
-    <tiptap-editor [(ngModel)]="content" [editor]="editor" (keyup)="subject.next()"></tiptap-editor>
+<div class="editor-wrapper" *ngIf="editor" [style]="customStyles">
+    <tiptap-editor [(ngModel)]="content" [editor]="editor"></tiptap-editor>
 </div>
 
 <dot-editor-count-bar
-    *ngIf="showCharData"
+    *ngIf="showCharData && editor"
```

Comment:
Since we are using the conditional `*ngIf="editor"` in both elements, we wrap them with an `ng-container` to use the conditional only once:

```html
<ng-container *ngIf="editor">
   <div>....</div>
   <dot-editor-count-bar>...</dot-editor-count-bar>
</ng-container>

```